omero/developers/Model.txt

```diff
@@ -129,17 +129,16 @@ include:
 -  details
 -  … any SQL keyword
 
-
 Properties
-~~~~~~~~~~
-Mutable, annotated, global.
+^^^^^^^^^^
 
+Mutable, annotated, global.
```

Comment:
It's for explanding. These are interfaces which need explaining. I'd think remove for the moment, and we can add back as necessary.